pom.xml

```diff
@@ -104,11 +104,6 @@
   </dependencyManagement>
 
   <dependencies>
-    <!-- TODO remove eddsa-api library ans us the new feature of Apache Mina SSH https://github.com/apache/mina-sshd/pull/639 -->
```

Comment:
Unfortunately, this won't be able to be done yet. 
At least until the plugin still supports the trilead library, or this will break users who haven't migrated and needs eddsa support.